.github/workflows/ci.yml

```diff
@@ -44,13 +30,15 @@ jobs:
     - name: Run tests
       run: |
         dotnet test
+        dotnet test -p:BouncyCastle=True # we did not run the non-Core tests on BouncyCastle previously
```

Comment:
nit: this comment might be better to put it on the commit message :)

Reply:
rather put a separate step? so that if it fails, it's more clear if the failure is BC or nonBC